python/samples/03.adaptiveCards.a.typeAheadBot/poetry.toml

```diff
@@ -0,0 +1,2 @@
+[virtualenvs]
```

Comment:
(In reference to TeamsChef-local screencap above)
Should this be updated to show TypeAhead bot name instead?

Reply:
this is just a poetry config file that tells it to use a local venv folder (like `node_modules`) instead of a global one.